### DISCUSSION TITLE
[BUG] ExemptFromBlockDownloadOfNonViewableFiles not working after setting the property

### DISCUSSION THREAD

**Discussion by potluri96** [pnp/powershell#2684] [General]
### Notice
Many bugs reported are actually related to the PnP Framework which is used behind the scenes. Consider carefully where to report an issue:

### Reporting an Issue or Missing Feature
Please confirm what it is that your reporting

I am trying to exempt a sharepoint library to allow downloads. In the SharePoint Admin Center, the unmanaged devices is set to "Allow Limited, Web-Only Access"

$list = Get-PnpList "Assets"; 
$list.SetExemptFromBlockDownloadOfNonViewableFiles($true) 
$list.Update()
Invoke-PnPQuery

### Expected behavior 
Please describe what output you expect to see from the PnP PowerShell Cmdlets

The warning banner will be gone for that library and the download button will be enabled.

### Actual behavior 
Please describe what you see instead. Please provide samples of output or screenshots.

The banner still shows up on the excluded library and the download button is not enabled.

### Steps to reproduce behavior
Please include complete script or code samples in-line or linked from [gists](https://gist.github.com/)

$list = Get-PnpList "Assets"; 
$list.SetExemptFromBlockDownloadOfNonViewableFiles($true) 
$list.Update()
Invoke-PnPQuery

### What is the version of the Cmdlet module you are running?
(you can retrieve this by executing ```Get-Module -Name "PnP.PowerShell" -ListAvailable```)

1.12.0

### Which operating system/environment are you running PnP PowerShell on?
- [ *] Windows
-
<img width="919" alt="Screenshot 2022-12-21 071436" src="https://user-images.githubusercontent.com/66481275/208913775-8ceaed03-fa38-4773-a4f6-f45f03e5dbe6.png">
<img width="466" alt="Screenshot 2022-12-21 071550" src="https://user-images.githubusercontent.com/66481275/208914002-9851799d-f25f-4438-bbeb-de141bfb1951.png">

**Comment by gautamdsheth** [comment]
The code is working fine, so it is not an issue with PnP PowerShell.
Moving this to discussion, seems like something else likely some configuration issue or maybe it takes some time. Could be something else as well.

**Comment by ghost** [comment]
Did you ever find an answer to this one, I have the exact same question/issue. Assuming it just sorted itself out after a few days?

**Reply by potluri96** [reply]
It’s working behind the scenes. The download button will not show up in the hi. When you run spfx webparts it’s working when you set the property on the clientsideassets library. My issue is resolved
